### DISCUSSION TITLE
Can't seem to format correctly?

### DISCUSSION THREAD

**Discussion by ngurney** [rianadon/timer-bar-card#118] [Q&A]
New to HA, but semi technical and I just can't figure this one out!! 

I have setup a dashboard using iOS themes, and everything styles and looks great except this one card (see screenshot)
![CleanShot 2023-12-31 at 10 04 40@2x](https://github.com/rianadon/timer-bar-card/assets/26093700/dfc23db0-3dd6-4d20-9af0-a098331aa45f)

Any ideas on how I can get this to take on the theme style?

**Comment by rianadon** [comment]
Hi! that's odd the card is displaying differently than the others but not unheard of. This may be similar to https://github.com/rianadon/timer-bar-card/issues/24. I thought I was using all the CSS classes Home Assistant expects, but maybe I missed something.

If you inspect element and hover over the backgrounds of one of the other cards, you should see hopefully see a `background` or `background-color` being set, and the tool will tell you which CSS class is setting that background. My guess is that the timer bar card is missing that class.

If you send me a link to the theme, I could also try doing the same on my end.

**Reply by ngurney** [reply]
These are the styles I am seeing with the thermostat card. 
![CleanShot 2024-01-06 at 12 52 57@2x](https://github.com/rianadon/timer-bar-card/assets/26093700/c1aa0272-d869-4f04-b1f7-04e13c8f0196)

This is the link to the theme: https://github.com/basnijholt/lovelace-ios-themes

I appreciate the response and help!

**Reply by rianadon** [reply]
I just had the time to play around with themes in my Home Assistant, and it looks like the card works with the iOS theme:

<img width="514" alt="image" src="https://github.com/rianadon/timer-bar-card/assets/18223213/c2259bce-dbac-4f6c-8795-fd5e2be0ef5e">

This is also what I have in my dev tools. The `timer-bar-card` element has an `ha-card` element inside, and it looks like that's where the theme styling is coming in.

<img width="345" alt="image" src="https://github.com/rianadon/timer-bar-card/assets/18223213/9ff25154-da7a-449d-9952-b3cd68954809">

Could you post your configuration? To be set up as a card, the config needs to have an `entities` option.

**Comment by scienceguyroy** [comment]
I have the same problem.  Here is what I'm seeing with the "graphite" theme:

![current view](https://github.com/user-attachments/assets/8d2fb3f4-05ba-4d85-8569-029adcb9c23c)

And here is my current configuration:

![configuration](https://github.com/user-attachments/assets/ae780a32-0fb1-4e32-915e-bc6ac186e700)

How do I get the Name font to be white?

Thanks!
